contract/test/proveEligibility.test.js

```diff
@@ -0,0 +1,250 @@
+// @ts-check
+// eslint-disable-next-line import/order
+import { test as anyTest } from './airdropData/prepare-test-env-ava.js';
+import { createRequire } from 'module';
+import path from 'path';
+import { E } from '@endo/far';
+import { AmountMath } from '@agoric/ertp/src/amountMath.js';
+import { TimeMath } from '@agoric/time';
+// ??? import { Id, IO, Task } from '../src/airdrop/adts/monads.js';
+import { bootAndInstallBundles } from './boot-tools.js';
+import { makeBundleCacheContext } from '../tools/bundle-tools.js';
+
+import '@agoric/store/exported.js';
+import { accounts, pubkeys, TEST_TREE_DATA } from './data/agoric.accounts.js';
+import { TimeIntervals } from '../src/airdrop/helpers/time.js';
+import { createDistributionConfig } from './utils.js';
+import { setup } from './setupBasicMints.js';
+
+const head = ([x]) => x;
+const parseAccountInfo = ({ pubkey, address }) => ({
+  pubkey: pubkey.key,
+  address,
+});
+
+const defaultClaimaint = {
+  // @ts-ignore
+  ...parseAccountInfo(head(accounts)),
+  proof: head(TEST_TREE_DATA.proofs),
+};
+const simulateClaim = async (
+  t,
+  invitation,
+  expectedPayout,
+  claimAccountDetails = defaultClaimaint,
+) => {
+  // claimAccountDetails object holds values that are passed into the offer as offerArgs
```

Comment:
@zmanian information about the claiming process. wall of this logic described here is to be implemented within `makeClaimInvitation` (linked to below)

 https://github.com/hindley-milner-systems/dapp-ertp-airdrop/blob/e55ee098942eefd14d18f3d7240bc1e4163781bf/contract/src/airdrop/prepare.js#L215-L234